swan-lake/learn/using-the-cli-tools.md

```diff
@@ -53,19 +54,18 @@ BALLERINA COMMANDS
    Core Commands:
         build           Compile Ballerina program into an executable
         run             Build and run Ballerina program
-        test            Run module tests
+        test            Run package tests
         doc             Generate API documentation
         clean           Clean artifacts generated during the build
         format          Format Ballerina sources
 
-   Module Commands:
-        pull            Pull a module from Ballerina Central
-        push            Upload module to the Ballerina Central
-        search          Search Ballerina Central for modules
-
-   Project Commands:
-        new             Create a new Ballerina project
-        add             Create a new Ballerina module in a project
+   Package Commands:
+        new             Create a new Ballerina package
+        init            Create a new Ballerina package in the current directory
```

Comment:
Can we use the word initialize here instead of create?

Reply:
This is a CLI output generated by the `bal help` tool. 
@praneesha WDYT about marking this as an improvement issue in the [ballerina-lang repo](https://github.com/ballerina-platform/ballerina-lang)?

Reply:
@keizer619 - WDYT?

Reply:
Will be address at https://github.com/ballerina-platform/ballerina-lang/issues/28138